lib/vagrant-service-manager/command.rb

```diff
@@ -183,7 +183,7 @@ def restart_service(service)
                     # to OpenShift service
                     "sccli #{service}"
```

Comment:
> Separate "generic services" from "core services"

So my understanding is currently sccli support core services (docker, openshift and kubernetes) and for generic we can directly use systemd. We are using sccli for services which require some configuration before starting it.